performance-test/README.md

```diff
@@ -0,0 +1,148 @@
+          mosip.esignet.cache.expire-in-seconds (authcodegenerated) - 21600
+          mosip.esignet.access-token-expire-seconds - 86400
+          mosip.esignet.id-token-expire-seconds - 86400
+          spring.cache.type=redis - check for this property and enable the redis.
+   * application default properties: Update the value for the below property.
+          mosip.kernel.otp.expiry-time - 86400
+   * id-authentication default properties: Update the value for the below properties.
+          otp.request.flooding.max-count - 100000
+
+* Create Resident Services EventId (Setup) : This thread contains service history endpoint api to capture the event id's required for the test.
+          
+
+### Data prerequisite
+
+* List of VID's as per environment which is valid and will be prepared from the above mentioned create identities setup.
+* List of Event id's as per environment which is valid and will be prepared from the above mentioned create resident services event id setup.
+
+### Execution points for Resident Service endpoints
+
+* The id and access token generated from the Login​ Redirect URI api will be stored in a file and will be used in the headers the api and can be re-used until they are not expired.
+* Test duration is already defined as a variable and needs to be changed as per the test run duration requirement in the user defined variable section.
+* The script is designed to run for 100 users, load being distributed accordingly for each thread group based on the weightage given for each scenario.
+
+
+### Exact steps of execution
+
+	Step 1: Enable only Create Identities in MOSIP Authentication System (Setup) thread group and toggle/disable the remaining thread groups in the script to create the required no of identities.
+	Step 2: Enable only Resident Id Access Token Creation (Setup) thread group and toggle/disable the remaining thread groups in the script to create access tokens and id tokens.
+	Step 3: Enable only Create Resident EventId (Setup) thread group and toggle/disable the remaining thread groups in the script to create the event id's.
+	Step 4: Enable the rest of all the Execution based scenario thread groups and toggle/disable the first 3 setup based thread groups. 
+	Step 5: Make sure test duration and ramp-up is defined in the user defined variable section. 
+	Step 5: Click on Run/Excute the test.
+	Step 6: Monitor the metrics during the test run using reports from jmeter, kibana, grafana and Jprofiler.
+	
+### Designing the workload model for performance test execution
+
+* Calculation of number of users depending on Transactions per second (TPS) provided by client
+
+* Applying little's law
+	* Users = TPS * (SLA of transaction + think time + pacing)
+	* TPS --> Transaction per second.
+
+* For the realistic approach we can keep (Think time + Pacing) = 1 second for API testing
+	* Calculating number of users for 10 TPS
+		* Users= 10 X (SLA of transaction + 1)
+		       = 10 X (1 + 1)
+			   = 20
```

Comment:
Is this pre configured as per the 100 tps in the scripts ? 
If yes, please add this as one liner into the readme also.

Reply:
Added in the read me file